crates/wots/src/lib.rs

```diff
@@ -0,0 +1,391 @@
+#![feature(generic_const_exprs)]
+#![allow(incomplete_features)]
+
```

Comment:
How this is different from what we already have in `main`: https://github.com/alpenlabs/strata-bridge/blob/2ff5d6cb95a5ea9de5caf386df9cbc99eb7631ab/crates/primitives/src/wots.rs ?

Reply:
We don't need another wots definition.
And the one on `main` is using `strata-bridge-primitives`.

Reply:
Ok, but shuould we move the signing functionality to the `primitives/src/wots.rs` and the feature gate `BitVM` in `strata-bridge-primitives`?
Cc @Rajil1213?

I don't like this code duplication.

Reply:
Yeah I just put that code into main. I agree we should factor everything out into a wots crate. in the mean time don't duplicate.

Reply:
Yeah I just put that code into main. I agree we should factor everything out into a wots crate. Still don't duplicate, though.

Reply:
https://github.com/BitVM/BitVM/pull/293 (@uncomputable changes) are already on `main`.